gtnh-modpack.json

```diff
@@ -984,6 +1020,18 @@
       "side": "BOTH",
       "maven": "http://jenkins.usrv.eu:8081/nexus/content/repositories/releases/com/github/GTNewHorizons/OpenModularTurrets/"
     },
+    {
+      "name": "Opis",
+      "version": "1.3.3-mapless",
+      "tagged_at": "2022-02-09T17:12:07",
+      "repo_url": "https://github.com/GTNewHorizons/Opis",
+      "filename": "Opis-1.7.10-1.3.3-mapless.jar",
+      "download_url": "https://api.github.com/repos/GTNewHorizons/Opis/releases/assets/56300556",
+      "browser_download_url": "https://github.com/GTNewHorizons/Opis/releases/download/1.3.3-mapless/Opis-1.7.10-1.3.3-mapless.jar",
+      "license": "CC BY-NC-SA 4.0",
+      "side": "BOTH",
```

Comment:
I don't think BOTH is accurate here...

Reply:
well it is: SP players want to be able to download opis for their SP map, server owners want them on their server, and server staff must have opis client side to use it.

Reply:
*Should* it though? Most people don't know it let alone know how to use it/what everything means. My "fear" is that it reduces the performance (if it does, if not everything is alright) for the benefit of a few.

Reply:
it's ok: opis does nothing until its lag engine is started with /opis

Reply:
It defaults to both.  This is the zip(s) that it will be included in.  Currently does nothing. Will need to update once we make the pack builder part.

Reply:
Also it defaults to BOTH when adding and we can change it later

I’m also tempted to at least make available the backported spark because trying to explain how to setup warmroast is an entire new level of difficulty heh

Reply:
could be a good addition yeah